content/fast/defer-unused-css/index.md

```diff
@@ -0,0 +1,149 @@
+       alt="DevTools performance trace for unoptimized page, showing FCP starting after CSS loads." class="screenshot">
+</figure>
+
+This means that the browser needs to wait for the whole CSS to be loaded and processed before painting a single pixel on the screen.
+
+## Optimize
+
+As a first step to optimize this page, you need to know which classes are actually considered “critical”.
+You'll use the the [Coverage Tool](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) for that:
+
+1. On DevTools, open the [Command Menu](https://developers.google.com/web/tools/chrome-devtools/ui#command-menu), by pressing Control+Shift+P or Cmd+Shift+P (Mac).
+1. Type “Coverage” and select “Show Coverage”.
+1. Click the “Reload” button, to reload the page and start capturing the coverage.
+
+<br/>
+
+<figure>
+  <img src="./coverage-unoptimized.png"
+       alt="Coverage for CSS file, showing 55.9% unused bytes." class="screenshot">
+</figure>
+
+
+Double-click on the report, to see classes marked in two colors:
+
+* Green (**"critical"**): These are the classes the browser needs to render the visible content (like the title, subtitle, and accordion buttons).
+* Red (**"non-critical"**): These styles are applied to content that's not immediately visible (like the paragraphs that appear after cliking on each button). Additionally, class definitions like "h2" are not even used on this page.
+
+With this information, you'll apply an optimization to let the browser start processing the “critical” styles immediately after the page loads, while deferring the load of the “non-critical” CSS for later:
+
+* Extract the class definitions marked with green on the report obtained from the coverage tool, and put those classes inside a ```<style>``` block on the head of the page:
+
+```
+<style type="text/css">
+.accordion-btn {background-color: #ADD8E6;color: #444;cursor: pointer;padding: 18px;width: 100%;border: none;text-align: left;outline: none;font-size: 15px;transition: 0.4s;}.container {padding: 0 18px;display: none;background-color: white;overflow: hidden;}h1 {word-spacing: 5px;color: blue;font-weight: bold;text-align: center;}
+</style>
+```
+
+* Then, load the rest of the classes asynchronously, by applying the following pattern:
+
+```
+<link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"> 
+<noscript><link rel="stylesheet" href="styles.css"></noscript>
+```
+
+This is not the standard way of loading CSS. Here's how it works:
+
+* ```link rel=”preload” as=”style”``` allows to request the style asynchronously (for more information about ```"preload"```  , check out [this guide](https://web.dev/fast/preload-critical-assets)).
+* The ```onload``` attribute on the link allows the CSS to be processed, when it finishes loading.
+* “nulling” the ```onload``` handler once it is used, helps some browsers to avoid re-calling the handler upon switching the rel attribute.
```

Comment:
```suggestion
* "nulling" the `onload` handler once it is used helps some browsers avoid re-calling the handler upon switching the rel attribute.
```

does switching the `rel` attribute happen that often? I don't think it's something I've manually done before.